dll/win32/browseui/explorerband.cpp

```diff
@@ -591,46 +524,176 @@ LRESULT CExplorerBand::OnShellEvent(UINT uMsg, WPARAM wParam, LPARAM lParam, BOO
     return 0;
 }
 
+BOOL
+CExplorerBand::IsTreeItemInEnum(
+    HTREEITEM hItem,
+    IEnumIDList *pEnum)
```

Comment:
SAL2 annotations for the new functions maybe?

Reply:
Fixed.